mantaray/backend.py

```diff
@@ -293,6 +300,14 @@ def _handle_received_message(self, msg: _ReceivedAndParsedMessage) -> None:
             self.event_queue.put(UserQuit(msg.sender, reason or None))
             return
 
+        if msg.command == "KICK":
+            assert msg.sender is not None
+            kicker = msg.sender
+            channel = msg.args[0]
+            kicked_nick = msg.args[1]
+            reason = msg.args[2]
```

Comment:
```suggestion
            channel, kicked_nick, reason = msg.args
```

Reply:
Ok, I didn't even know it could be done this way